cmd/idracsdkapps/splunkapp/main.go

```diff
@@ -263,13 +280,13 @@ func main() {
 
 	for {
 
-		req, err := http.NewRequest("GET", "https://10.239.37.136/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
+		req, err := http.NewRequest("GET", "https://100.69.104.132/redfish/v1/SSE?$filter=EventFormatType%20eq%20MetricReport", nil)
```

Comment:
If I remember correctly this code will run on the iDRAC.  The HTTP Request can be changed to target the iDRAC it is on with a switch from 100.69.104.132 to "localhost"